torch/_dynamo/variables/lists.py

```diff
@@ -208,6 +209,119 @@ def var_getattr(self, tx, name):
         return self.items[fields.index(name)].add_options(self)
 
 
+def call_common_append(
```

Comment:
these should all be code-motion, only diff is swapping `ListVariable` for `variable_class` so it can be shared with DequeVariable

Reply:
Perhaps a common base class?  Is this everything or are you skipping some?

Reply:
yea.. i almost had Deque extend from List.

I felt like fundamentally a Deque isn't actually a customized list, so maybe that wasn't the right thing to do.

I could also put another layer inbetween them.